ui/apps/platform/src/Containers/Vulnerabilities/VulnerablityReporting/ModifyVulnReport/CloneVulnReportPage.tsx

```diff
@@ -1,7 +1,163 @@
-import React from 'react';
+import React, { useEffect } from 'react';
+import { useHistory, useParams } from 'react-router-dom';
+import {
+    PageSection,
+    Title,
+    Divider,
+    Flex,
+    FlexItem,
+    Breadcrumb,
+    BreadcrumbItem,
+    Wizard,
+    Bullseye,
+    Spinner,
+} from '@patternfly/react-core';
+
+import { vulnerabilityReportsPath } from 'routePaths';
+import useReportFormValues from 'Containers/Vulnerabilities/VulnerablityReporting/forms/useReportFormValues';
+import useCreateReport from 'Containers/Vulnerabilities/VulnerablityReporting/api/useCreateReport';
+import useFetchReport from 'Containers/Vulnerabilities/VulnerablityReporting/api/useFetchReport';
+
+import PageTitle from 'Components/PageTitle';
+import BreadcrumbItemLink from 'Components/BreadcrumbItemLink';
+import ReportParametersForm from 'Containers/Vulnerabilities/VulnerablityReporting/forms/ReportParametersForm';
+import DeliveryDestinationsForm from 'Containers/Vulnerabilities/VulnerablityReporting/forms/DeliveryDestinationsForm';
+import ReportReviewForm from 'Containers/Vulnerabilities/VulnerablityReporting/forms/ReportReviewForm';
+import NotFoundMessage from 'Components/NotFoundMessage/NotFoundMessage';
+import { getReportFormValuesFromConfiguration } from '../utils';
+import ReportFormErrorAlert from './ReportFormErrorAlert';
+import ReportFormWizardFooter from './ReportFormWizardFooter';
+
+const wizardStepNames = [
+    'Configure report parameters',
+    'Configure delivery destinations (Optional)',
+    'Review and create',
+];
 
 function CloneVulnReportPage() {
-    return <div />;
+    const history = useHistory();
+    const { reportId } = useParams();
+
+    const { reportConfiguration, isLoading, error } = useFetchReport(reportId);
+    const { formValues, setFormValues, setFormFieldValue, clearFormValues } = useReportFormValues();
+    const { data, isLoading: isCreating, error: createError, createReport } = useCreateReport();
+
+    // When report is created, navigate to the vuln reports page
+    // @TODO: We want to change this in the future to navigate to the read-only report page to verify the details
+    useEffect(() => {
+        if (data) {
+            clearFormValues();
+            history.push(vulnerabilityReportsPath);
+        }
+        // eslint-disable-next-line react-hooks/exhaustive-deps
+    }, [data]);
```

Comment:
Same nit as the parent branch: https://github.com/stackrox/stackrox/pull/7101#discussion_r1273431672